doc/harmony-ap_logging_user_guide.md

```diff
@@ -0,0 +1,371 @@
+- `business.log`: This log file contains all the business-related information. For example, when a message is sent or received, etc.
+- `domibus.log`: This log file contains both the security and business logs plus miscellaneous logs like debug information, logs from one of the framework used by the application, etc.
+- `domibus-error.log`: This log file contains all the errors which occurred in Domibus including errors from third party libraries used by Domibus.
+- `statistics.log`: This log file includes information on the occurrence of different events (receive message, submit message, etc.).
+
+#### 3.1.2 Modifying the Logback configuration
+
+Logback configuration in the Harmony AP Linux package is located at `/etc/harmony-ap/logback.xml`. To apply changes:
+
+1. Edit the `logback.xml` file on the host machine.
+2. Save the changes and restart the Harmony AP service:
+   ```bash
+   systemctl restart harmony-ap
+   ```
+
+#### 3.1.3 Log rotation
+
+Log rotation is the process of managing log files to prevent them from becoming too large and consuming too much disk space. Log rotation is essential to ensure that log files are manageable and that the system does not run out of disk space.
+
+By default, AP linux package uses the same configuration for all the logs saved into files:
+
+- Maximum file size: Each log file can grow up to 30MB before a new file is created.
+- Maximum history: Up to 60 days worth of log files are retained.
+- Total size cap: The total size of all log files is capped at 20GB. If this limit is reached, older files are deleted to make room for new ones, independently of the maximum history setting.
+
+The following example shows the default configuration for log rotation:
+
+```xml
+<rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+  <fileNamePattern>${logdir}/statistics-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+  <maxFileSize>30MB</maxFileSize>
+  <maxHistory>60</maxHistory>
+  <totalSizeCap>20GB</totalSizeCap>
+</rollingPolicy>
+```
+
+### 3.2 Logback configuration in Harmony AP Docker image
+
+The default configuration file for Logback is available in the [logback.xml](https://github.com/nordic-institute/harmony-common/blob/main/packaging/ap/docker/logback.xml).
+
+#### 3.2.1 Viewing logs
+
+Logs are sent to the standard output (`stdout`), this allows the logs to be captured by Docker and viewed using the `docker logs` command:
+
+```bash
+docker logs -f <container-name>
+```
+
+Since Docker handles log files, Logback does not save logs to the file system.
+
+#### 3.2.2 Modifying the Logback configuration
+
+Logback configuration in the Harmony AP Linux package is located at `/var/opt/harmony-ap/etc/logback.xml`. To modify the Logback configuration in the Harmony AP Docker image, create a custom image with the modified configuration:
```

Comment:
I think it would be good to mention here the advanced configuration approach explained in the [Docker Installation Guide](https://github.com/nordic-institute/harmony-common/blob/develop/doc/harmony-ap_docker_installation_guide.md#322-advanced-configuration) since it enables customising the configuration without having to create a custom image.